components/server/typings/express/index.d.ts

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
This file is a duplicate of the one at src/express.ts.

Can we reference that one instead? Or avoid the duplication any other way? :thinking:

Reply:
it should be deleted. missed that when splitting up a larger pr